README.md

```diff
@@ -70,11 +70,13 @@ Governor.prototype.wrap = fn => {
 ```
 
 Similarly, `wrapIterator(it: Iterator<T> | AsyncIterator<T>): AsyncIterator<T>` takes an Iterator or AsyncIterator and returns an AsyncIterator that yields the same values but limited in concurrency by this Governor.
+
+There are also static helpers for composing Governors: `Governor.any` and `Governor.all`. `any` takes 0 or more Governors and produces a Governor that, when acquired, attempts to acquire all of the passed Governors, returns the first GovernorToken it receives, and releases any other tokens it acquires. `all` takes 0 or more Governors and produces a Governor that attempts to acquire all of the passed Governors and only returns a GovernorToken once it has received a token for each of them.
```

Comment:
why would it make sense to take zero governors?

Reply:
When you're dynamically managing a list of dependencies and that list ends up being empty.

Reply:
but in that case you'd get no governor token at all. why is that more useful than a rejection/exception?

Reply:
You do get a token though. Take a look at the code.

Reply:
i'm just reading the prose:

> returns the first GovernorToken it receives, and releases any other tokens it acquires

if it receives none, how can it return one?

> and only returns a GovernorToken once it has received a token for each of them.

i guess i can see this as a case of `[].every(() => true)`, but while logical, the behavior of some/every on empty arrays is pretty confusing for most.

Reply:
`all` must give you a token right away when given an empty list. Compare `Promise.all`, which gives you a settled promise right away.

IMO `any` should throw when given an empty list. Again, compare `Promise.any`. Alternatively it could never resolve, like how `AbortSignal.any([])` never aborts, but that seems worse; I don't like producing never-settling promises. Giving you a token right away like `all` does is wrong, both theoretically (you can't get 'any' item an empty list) and practically (if you're waiting for any one of N threads to be available for your use, and N is 0, you have not successfully waited for a thread and this API shouldn't suggest you have).

Reply:
Alright fine. A promise that never resolves seems like it's more consistent, but throwing immediately seems more practically desirable.